examples/_includes/macros/tab/styles.xml.njk

```diff
@@ -1,2 +1,35 @@
 <style id="tab-content" flex="1" />
 <style id="centered" alignItems="center" justifyContent="center" />
+<style
+  id="row"
```

Comment:
we should use a prefix for all the styles so there's no risk of collision with other styles
```suggestion
  id="tab-row"
```